ingestion_tools/scripts/dataset_config_validate.py

```diff
@@ -36,24 +36,18 @@
 INTEGER_FORMATTED_STRING_REGEX = r"^int\s*{[a-zA-Z0-9_-]+}\s*$"
 
 
-def _error_to_filtered_text(error_value: Union[dict, str]) -> str:
+def print_errors(errors: Dict[str, List[Union[ValidationError, Exception]]]) -> List[str]:
     """
-    Convert an error value to a filtered text string (for easier comparison and filtering of errors)
+    Convert an error value to a more concise text string.
     """
-    if not isinstance(error_value, dict) or any(key not in error_value for key in ["loc", "msg", "input"]):
-        return str(error_value)
-    loc = "/".join([str(x) for x in error_value["loc"]])
-    return f"{loc}: {error_value['msg']} (Input: {error_value['input']})"
-
-
-def _error_to_filtered_2_text(error_value: Union[dict, str]) -> str:
-    """
-    Convert an error value to a filtered text string (for easier comparison and filtering of errors)
-    This one removes more attributes from the error value, so that the errors list can be filtered down even more.
-    """
-    if not isinstance(error_value, dict) or any(key not in error_value for key in ["loc", "msg"]):
-        return str(error_value)
-    return str(error_value["loc"][-1]) + ": " + str(error_value["msg"])
+    for file, error in sorted(errors.items()):
+        print(f'FAIL: "{file}":')
+        for e in error:
+            if not isinstance(e, dict) or any(key not in e for key in ["loc", "msg"]):
+                print("\t- error: " + str(e))
+            else:
+                loc = "/".join([str(x) for x in e["loc"]])
```

Comment:
nit: I'd prefer the separator to be a `.` instead of a `/` since it's a more common object traversal notation